DashSync/shared/Models/DAPI/DSDAPIClient.m

```diff
@@ -308,9 +308,9 @@ - (void)publishTransition:(DSTransition *)stateTransition
           completionQueue:(dispatch_queue_t)completionQueue
                   success:(void (^)(NSDictionary *successDictionary, BOOL added))success
                   failure:(void (^)(NSError *error))failure {
-    //default to 5 attempts
+    //default to 10 attempts
     [self publishTransition:stateTransition
-                 retryCount:5
+                 retryCount:10
```

Comment:
I would suggest we set this as a default const value somwhere.

Reply:
Introduced new constant